java/README.md

```diff
@@ -2,57 +2,58 @@
 
 ## Verification Task Structure
 
-Verification tasks are grouped in directories, e.g.,
-`jbmc-regression`.  Within these directories, each verification task
-is in its own directory, e.g., `StringValueOf01_true-assert`. The
-suffix of this directory contains the expected verification outcome,
-i.e., `_true-assert` means that there is no execution of the program
-that violates any of the assertions in that program.  A verification
-task has a structure compatible with the [Maven](https://maven.apache.org)
-standard directory layout (to allow the future use of the Maven build
-system if necessary):
-
-| directory | description |
-| --------- | ----------- |
-|`src/` | |
-| `src/main/java/Main.java` | The `Main` class. |
```

Comment:
The info on the project structure is not obsolete.

Reply:
I really don't want to have to maintain a different directory structure for each benchmark. Let's stick to something that is standard.

Reply:
But the new proposal would actually simplify this maintenance: Instead of having to check for and enforce a specific directory structure and implementing assumptions about this directory structure in tools, there would just be a list of path for input files (written down explicitly in the YAML file and passed to the verifier).

What advantage do we get from forcing everything to be in `src/main/java`? For what do we need `target`?

Reply:
I think we can get rid of `src/main/java`, but should keep the guideline that the source tree starts at the root of the benchmark dir. Also, we have to keep the assumption that there is a `Main` class in the root package with a `main` method because this is currently hard-coded in the `assertion.prp`.